component/azstorage/azstorage.go

```diff
@@ -539,6 +539,10 @@ func (az *AzStorage) GetAttr(options internal.GetAttrOptions) (attr *internal.Ob
 	return az.storage.GetAttr(options.Name)
 }
 
+func (az *AzStorage) SetAttr(options internal.SetAttrOptions) error {
+	return az.storage.SetAttr(options.Name, options.Attr)
+}
+
```

Comment:
do you want to move changes to azstorage folder, to another PR, like you had done for other such changes, like read-with-filename and conditional creation of file/folder.